pipeline/outputs/s3.md

```diff
@@ -18,31 +18,31 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
+| sts_endpoint                  | Custom endpoint for the STS API.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | None                                    |
+| canned_acl                    | [Predefined Canned ACL policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl) for S3 objects.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | None                                    |
+| compression                   | Compression type for S3 objects. 'gzip' is currently the only supported value. The Content-Encoding HTTP Header will be set to 'gzip'. Compression all type can be enabled when `use_put_object` is on, Compression Gzip can be enabled when `use_put_object` on or off. If Apache Arrow support was enabled at compile time, you can set 'arrow' to this option.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | None                                    |
```

Comment:
let's change this to something like: 

"Compression type for S3 objects. 'gzip' is currently the only supported value by default. The Content-Encoding HTTP Header will be set to 'gzip'.  If Apache Arrow support was enabled at compile time, you can set 'arrow' to this option. Compression 'gzip' can be enabled with both `use_put_object` on or off. Apache 'arrow' compression can only be enabled with use_put_object` on." 

I think this is more readable.

Reply:
Will do